src/shared/DotnetTool/pack-tool.sh

```diff
@@ -38,7 +37,7 @@ esac
 done
 
 if [ -z "$VERSION" ]; then
-    VERSION="$GitBuildVersionSimple"
```

Comment:
`$GitBuildVersionSimple` isn't set in dev builds, so let's just require it and have the CI pass this in. If `$VERSION` ends up being empty the dev builds ended up half-baked.